packages/wrangler/src/publish.ts

```diff
@@ -102,14 +102,36 @@ export default async function publish(props: Props): Promise<void> {
       }
     );
 
+    // Some validation of durable objects + migrations
+    if (config.durable_objects.bindings.length > 0) {
+      // TODO: implement durable objects for service environments
+      if (!props.legacyEnv) {
+        throw new Error(
+          "Publishing Durable Objects to a service environment is not currently supported. We'll fix this soon!"
```

Comment:
"We'll fix this soon" is not very actionable or particularly helpful.
How about we link to an issue where this is being tracked, and the user can add their use-case?

```suggestion
          "Publishing Durable Objects to a service environment is not currently supported. This is being tracked at <some-github-url>"
```